samples/dotnet/KernelHttpServer/Config/Constants.cs

```diff
@@ -8,12 +8,12 @@ public class SKHttpHeaders
     {
         public const string CompletionModel = "x-ms-sk-completion-model";
         public const string CompletionEndpoint = "x-ms-sk-completion-endpoint";
-        public const string CompletionBackend = "x-ms-sk-completion-backend";
+        public const string CompletionService = "x-ms-sk-completion-backend";
```

Comment:
Should we update also header name? (e.g. `x-ms-sk-completion-service` and `x-ms-sk-embedding-service`.

Reply:
something for later probably. FYI @craigomatic I left this out because it will affect all the apps and this PR is already pretty big

Reply:
I'd say we leave this for later